python/cuml/cuml/tests/test_meta_estimators.py

```diff
@@ -35,7 +35,7 @@ def test_pipeline():
     pipe = Pipeline(steps=[("scaler", StandardScaler()), ("svc", SVC())])
     pipe.fit(X_train, y_train)
     score = pipe.score(X_test, y_test)
-    assert score > 0.8
+    assert score > 0.75
```

Comment:
`score` before this change was like `0.80000001`, it's now `0.8` (the joys of floating point). Really this test shouldn't care about the exact score, it's more a test that everything is plumbed together correctly.

Reply:
I guess the question is what this test is testing. The [new test for `accuracy_score`](https://github.com/rapidsai/cuml/pull/6406/files#diff-21d3ff6d3c9e67417fe1c52384123630490a979a65aedfdd3b15288ef4b4a7c3R202) is much stricter and more explicit than it used to be. It checks that the cuml result is `np.allclose` to the scikit-learn result (without some intermediate estimator fitting).

So I think this test is mostly checking that things are wired up correctly and don't crash when you use them. A score above 0.5 is probably what we'd hope for, below that something is weird, though unlikely that it would be caused by `Pipeline`.